q2cli/tests/test_cli.py

```diff
@@ -66,6 +66,18 @@ def test_plugin_list_commands(self):
         self.assertFalse('split_ints' in commands)
         self.assertFalse('mapping_viz' in commands)
 
+    def test_show_importable_types(self):
+        result = self.runner.invoke(
+            tools, ['import', '--show-importable-types'])
+        self.assertEqual(result.exit_code, 0)
+        self.assertTrue('IntSequence1' in result.output)
```

Comment:
Got it.  Thanks @jairideout !  This has been a great learning experience  😃  

Improving my knowledge of Git and learning about Travis CI was super valuable!